content/applications/inventory_and_mrp/inventory/routes/concepts/stock_warehouses.rst

```diff
@@ -10,187 +10,220 @@ In Odoo, sometimes products included in a single sales order might take stock fr
+      :alt: Graphic of child locations 'WHA' and 'WHB' tied to the parent location.
+
+.. _inventory/routes/link-to-vwh:
+
+Link child warehouses to virtual stock
+======================================
+
+To set physical warehouses as child locations of the virtual location configured in the
+:ref:`previous step <inventory/routes/virtual-wh>`, navigate to :menuselection:`Inventory app -->
+Configuration --> Locations`.
+
+Remove any filters from the search bar. Then, click the physical warehouse :guilabel:`Location` that
+was previously created to be a child location (e.g. `WHA`), and click :guilabel:`Edit`.
+
+Change the :guilabel:`Parent Location` field from :guilabel:`Physical Locations` to the virtual
+warehouse's **stock location** (e.g. `VWH/Stock`) from the drop-down menu, and click
+:guilabel:`Save`.
 
 .. important::
-   In order to take stock from multiple warehouses to fulfill a sales order, there need to be at
-   least **two** warehouses acting as *child locations* of the *virtual parent location* warehouse.
+   To select the virtual warehouse's stock location in the :guilabel:`Parent Location` drop-down
+   menu, the parent warehouse stock location (e.g. `VWH/Stock`) **must**  have its
+   :guilabel:`Location Type` set to :guilabel:`Internal Location`.
 
-To create and edit *Locations*, go to :menuselection:`Inventory --> Configuration --> Locations`.
-All :guilabel:`Locations` are listed here, including the *Stock* :guilabel:`Location` of the virtual
-warehouse that was created. Click into the *Stock* :guilabel:`Location` for the virtual warehouse
-that was previously created (:dfn:`Warehouse Name/Stock`).
+.. image:: stock_warehouses/configure-physical-wh.png
+   :align: center
+   :alt: Set the child warehouse's *Parent Location* to the virtual warehouse.
 
-Then, under the :guilabel:`Additional Information` section, change the :guilabel:`Location Type`
-from :guilabel:`Internal Location` to :guilabel:`View`. :guilabel:`Save` the changes.
+Repeat the preceding steps to configure two or more child warehouses.
 
-This identifies this :guilabel:`Location` as a *virtual location*, which is used to create a
-hierarchical structure for a warehouse and aggregate its *child locations*.
+Once complete, the virtual, parent warehouse (e.g. `VWH/Stock`) fulfills orders using stock from
+child warehouses (e.g. `WHA` and `WHB`), if there is insufficient stock in any one location.
 
-.. note::
-   Products can *not* be stored in a :guilabel:`View` :guilabel:`Location Type`.
+Set virtual stock location as 'view'
+====================================
+
+.. tip::
+   Change the virtual stock location to :guilabel:`View` to create a hierarchical structure for a
+   warehouse. Doing so displays the total on-hand quantity of the product across **all** linked
+   child warehouses.
+
+   To view the quantity of a product across all linked child warehouses, go to the product form and
+   click the :guilabel:`On Hand` smart button.
```

Comment:
```suggestion
   Optional: Set the virtual stock location's :guilabel:`Location Type` to :guilabel:`View` since it is a non-existent location used to group various physical warehouses together.
   
.. tip::

   To view the quantity of a product across **all** linked child warehouses, go to the product form and
   click the :guilabel:`On Hand` smart button.
```